redis-persistence/src/main/java/com/netflix/conductor/dao/dynomite/queue/DynoQueueDAO.java

```diff
@@ -150,10 +152,15 @@ public boolean pushIfNotExists(String queueName, String id, long offsetTimeInSec
 
     @Override
     public List<String> pop(String queueName, int count, int timeout) {
-        List<Message> msg = queues.get(queueName).pop(count, timeout, TimeUnit.MILLISECONDS);
-        return msg.stream()
-				.map(Message::getId)
-				.collect(Collectors.toList());
+        try {
+            List<Message> msg = queues.get(queueName).pop(count, timeout, TimeUnit.MILLISECONDS);
+            return msg.stream()
+                    .map(Message::getId)
+                    .collect(Collectors.toList());
+        } catch (NoAvailableHostsException ex) {
```

Comment:
The underlying RedisDynoQueue from dyno-queues implementation wraps NoAvailableHostsException in RuntimeException. Can we still catch it here?